samples/testapp/src/commonMain/composeResources/values/strings.xml

```diff
@@ -16,6 +16,8 @@
     <string name="iso_mdoc_proximity_sharing_title">ISO mdoc Proximity Sharing</string>
     <string name="iso_mdoc_proximity_reading_title">ISO mdoc Proximity Reading</string>
     <string name="iso_mdoc_multi_device_testing_title">ISO mdoc Multi-Device Testing</string>
+    <string name="certificates_viewer_title">CertificatesViewer Examples</string>
+    <string name="x509_cert_chain_viewer_title">X.509 Cert Chain Viewer</string>
```

Comment:
Nit: I would rename these to string identifiers to `certificate_viewer_examples_title` and `certificate_viewer_title`.